app/components/subcase/subcase-description-panel/subcase-description-edit.js

```diff
@@ -43,12 +44,15 @@ export default class SubcaseDescriptionEdit extends Component {
   @task
   *loadAgendaItemTypes() {
     this.agendaItemTypes = yield this.store.query('concept', {
-      sort: '-label',
       filter: {
         'concept-schemes': {
           ':uri:': CONSTANTS.CONCEPT_SCHEMES.AGENDA_ITEM_TYPES,
         }
       },
+      page: {
+        size: PAGE_SIZE.CODE_LISTS,
+      },
+      sort: '-label',
```

Comment:
Why not using the `queryConceptsForConceptScheme` method here? Because of the different sorting? Can we make the method accept a second `options` argument in which you can overwrite the `sort` param?